backend/webserver/models.py

```diff
@@ -57,4 +58,12 @@ class Comment(models.Model):
 class Like(models.Model):
     author = models.ForeignKey(Author,on_delete=models.CASCADE)
     post = models.ForeignKey(Post,on_delete=models.CASCADE,null=True)
-    comment = models.ForeignKey(Comment,on_delete=models.CASCADE,null=True)
\ No newline at end of file
+    comment = models.ForeignKey(Comment,on_delete=models.CASCADE,null=True)
+
+class Inbox(models.Model):
+    author= models.ForeignKey(Author,on_delete=models.CASCADE)
+    accepted = models.ForeignKey(Author, related_name='following_requests_accepted', on_delete=models.CASCADE)
+    requested = models.ForeignKey(Author, related_name='inbox_requests_received', on_delete=models.CASCADE)
```

Comment:
```suggestion
    follow_request_sender = models.ForeignKey(Author, related_name='sent_follow_requests', on_delete=models.CASCADE, verbose_name='author who sent the follow request', null=True)
```
- `null=True` because an Inbox row could be unrelated to a follow request